monitoring/dashboard.py

```diff
@@ -630,6 +819,18 @@ def top10_errors_by_bucket(title, code):
                 top10Error500ByBucket,
                 top10Error5xxByBucket
             ], height=8),
+            RowPanel(title="Quotas"),
+            layout.row([
+                layout.resize([quotaHealth], width=6),
+                layout.resize([quotaStatusCode], width=6),
+                quotaByAction,
+            ], height=8),
+            layout.row([
+                layout.resize([bucketQuotaCounter], width=3),
+                layout.resize([accountQuotaCounter], width=3),
```

Comment:
we probably want to stack these on top of each other, will look better (they look quite stretched, due to the small width & large height)